src/test/ui/array-slice-vec/vec-to_str.rs

```diff
@@ -2,11 +2,6 @@
 
 
 pub fn main() {
-    assert_eq!(format!("{:?}", vec![0, 1]), "[0, 1]".to_string());
-
-    let foo = vec![3, 4];
     let bar: &[isize] = &[4, 5];
-
-    assert_eq!(format!("{:?}", foo), "[3, 4]");
     assert_eq!(format!("{:?}", bar), "[4, 5]");
 }
```

Comment:
Hm, why is this file not removed completely? Or is it a bug in a way github displays the diff?

Reply:
I left the file, because the remaining parts were the ones testing a raw slice, not a vector. I wasn't sure if there was a place to put unit tests of the built-in slice type, and didn't want to remove a possibly useful test

Reply:
Ah, I understand. While raw slice is indeed built-in, its pretty shallow in a sense that most of the slice API are implemented in libcore, and not directly in the compiler. For example, this code is implemented in core/fmt:

https://github.com/rust-lang/rust/blob/3edf11cf811707cde49b30a80896c24189649d95/library/core/src/fmt/mod.rs#L2172-L2177

So, I think we also should move this over to `test_debug_fmt`.

Ideally, we probably should move this to the rest of formatting tests, but they are also a run-pass now (called `ifmt` for some obscure historical reason), and moving that goes out of scope for this PR.

Reply:
Alright, I'll move it with vec for now and delete the file. Future work can hopefully clean-up the formatting tests

Reply:
JFYI, moved some of fmt tests to more sane location in #76301